.github/workflows/sync_translations.yml

```diff
@@ -36,12 +37,11 @@ jobs:
         uses: crowdin/github-action@1.4.8
         env:
           GITHUB_TOKEN: ${{secrets.OTTO_THE_BOT_GH_TOKEN}}
+          GITHUB_ACTOR: otto-the-bot
           CROWDIN_PROJECT_ID: 342359
           INPUT_DEBUG_MODE: false
         with:
           upload_sources: true
-          github_user_name: otto-the-bot
-          github_user_email: webapp+otto@wire.com
```

Comment:
A possible explanation for our translation update jobs to fail, would be that this user is not identified as a contributor with write access to the webapp repo... :shrug:

Reply:
i tried to switch it to otto before because it does have write access 😬